test/e2e/setup.js

```diff
@@ -1,4 +1,8 @@
-module.exports.start = function () {
+module.exports = {
+  loraOSConfigured: false
+}
+module.exports.start = function (test) {
+  let me = this
   return new Promise(async function (resolve, reject) {
     resolve()
   })
```

Comment:
You can use `return Promise.resolve()` here

Reply:
Its a place holder anyway.  But thanks